src/commands/init.ts

```diff
@@ -112,14 +112,21 @@ initCommand.action(async () => {
           title: "Updating Claude Desktop settings",
           enabled: (ctx) => ctx.claudeDesktopInstalled && !!ctx.accessToken,
           task: async (ctx, configTask) => {
-            const configPath = getClaudeConfigPath();
+            const configFolderPath = getClaudeConfigFolderPath();
+            const configFilePath = join(configFolderPath, "claude_desktop_config.json");
             const npxPath = await getNpxPath();
 
-            if (!fs.existsSync(configPath)) {
-              throw new Error(`Can't find Claude's config file at: ${configPath}`);
+            if (!fs.existsSync(configFolderPath)) {
+              throw new Error(
+                "I couldn't find Claude Desktop on your system. You'll need to install it first, or follow the manual setup instructions if you're using a different MCP-compatible app.",
+              );
+            }
+
+            if (!fs.existsSync(configFilePath)) {
+              fs.writeFileSync(configFilePath, JSON.stringify({}, null, 2));
```

Comment:
I think we can keep one `writeFileSync` by creating/writing to the file at the end - line 145, and here just construct the object.